src/synth.rs

```diff
@@ -54,20 +56,20 @@ pub fn try_init(
     if !syn.set_polyphony(16) {
         warn!("Failed to set polyphony to 16");
     }
-    
+
```

Comment:
There's a few whitespace changes, just from running `rustfmt`. I can undo these if you prefer

Reply:
Nah, that's fine.